launcher_scripts/conf/training/gpt3/175b.yaml

```diff
@@ -117,13 +117,16 @@ model:
   fp8_amax_history_len: 1 # Number of steps for which amax history is recorded per tensor
   fp8_amax_compute_algo: most_recent # 'most_recent' or 'max'. Algorithm for computing amax from history
   use_emha: False
+  ub_tp_comm_overlap: True
 
   # miscellaneous
   seed: 1234
   sync_batch_comm: False
   use_cpu_initialization: False # Init weights on the CPU (slow for large models)
   onnx_safe: False # Use work-arounds for known problems with Torch ONNX exporter.
   apex_transformer_log_level: 30 # Python logging level displays logs with severity greater than or equal to this
+  overlap_p2p_comm: True # Overlap p2p communication with computes
+  batch_p2p_comm: False # Batch consecutive inter-peer send/recv operations
```

Comment:
can you add the `gc_interval` config as well?

Reply:
also this config isn't using transformer engine

Reply:
updated all configs